src/MongoDB.Driver.Core/Core/Operations/AggregateOperation.cs

```diff
@@ -35,7 +35,7 @@ namespace MongoDB.Driver.Core.Operations
     /// Represents an aggregate operation.
     /// </summary>
     /// <typeparam name="TResult">The type of the result values.</typeparam>
-    public class AggregateOperation<TResult> : IReadOperation<IAsyncCursor<TResult>>
+    public class AggregateOperation<TResult> : IReadOperation<IAsyncCursor<TResult>>, IExecutableInRetryableReadContext<IAsyncCursor<TResult>>
```

Comment:
Note: AggregateOperation itself is not a IRetryableReadOperation because it doesn't need to be. It ends up creating a ReadCommandOperation to delegate execution of the aggregate command to.

Note: ReadCommandOperation *is* a IRetryableReadOperation!